File: jb/src/main/java/com/codestream/language/NrqlTypes.java
package com.codestream.language;

import com.intellij.psi.tree.IElementType;

public interface NrqlTypes {
    public IElementType OPERATOR = new NrqlTokenType("OPERATOR");
    public IElementType ATTRIBUTE_NAME = new NrqlTokenType("ATTRIBUTE_NAME");
    public IElementType NUMBER = new NrqlTokenType("NUMBER");
    public IElementType SINGLE_QUOTED_STRING = new NrqlTokenType("SINGLE_QUOTED_STRING");
    public IElementType COMMENT = new NrqlTokenType("COMMENT");
    public IElementType KEYWORD = new NrqlTokenType("KEYWORD");
    public IElementType PREPROCESSOR = new NrqlTokenType("PREPROCESSOR");
    public IElementType FUNCTION = new NrqlTokenType("FUNCTION");
    public IElementType DOUBLE_QUOTED_STRING = new NrqlTokenType("DOUBLE_QUOTED_STRING");
}

File: jb/src/main/kotlin/language/NrqlTypes.kt
package com.codestream.language

public class NrqlTypesz {
    public companion object {
        public val OPERATOR = NrqlTokenType("OPERATOR")
        val ATTRIBUTE_NAME = NrqlTokenType("ATTRIBUTE_NAME")
        val SINGLE_QUOTED_STRING = NrqlTokenType("SINGLE_QUOTED_STRING")
        val COMMENT = NrqlTokenType("COMMENT")
        val KEYWORD = NrqlTokenType("KEYWORD")
        val PREPROCESSOR = NrqlTokenType("PREPROCESSOR")
        val FUNCTION = NrqlTokenType("FUNCTION")
        val DOUBLE_QUOTED_STRING = NrqlTokenType("DOUBLE_QUOTED_STRING")
    }
}
